### PR TITLE
UI: Create Password

### PR DESCRIPTION
**Pull request by dvaldivia**
Screen Shot 2020-01-31 at 4 09 45 PM

### CODE REVIEW

**Review by Alevsk — APPROVED** [review]

File: portal-ui/src/screens/CreatePassword/index.tsx [hunk 1]
```diff
@@ -0,0 +1,269 @@
...
+                setError(err);
+            });
+    }
+
+    // validate the token passed via url
+
+    React.useEffect(() => {
+        const urlParams = new URLSearchParams(window.location.search);
+        const urlToken = urlParams.get("t") as string;
+        if (urlToken == null || urlToken == '') {
```
Comment (kaankabalak) [inline portal-ui/src/screens/CreatePassword/index.tsx:115]: Can we use triple equals here? We are getting a warning here:

```
  Line 115:42:  Expected '===' and instead saw '=='  eqeqeq
```

```suggestion
        if (urlToken === null || urlToken === '') {
```

File: portal-ui/src/screens/CreatePassword/index.tsx [hunk 2]
```diff
@@ -0,0 +1,269 @@
...
+    const [token, setToken] = React.useState<string>('');
+    const [tokenValidated, setTokenValidated] = React.useState<boolean>(false);
+    const [tokenValid, setTokenValid] = React.useState<boolean>(true);
+    const [repeatPassword, setRepeatPassword] = React.useState<string>('');
+    const [error, setError] = React.useState<string>('');
+
+    const formSubmit = (e: React.FormEvent<HTMLFormElement>) => {
+        e.preventDefault();
+
+        if (password != repeatPassword) {
```
Comment (kaankabalak) [inline portal-ui/src/screens/CreatePassword/index.tsx:78]: Similarly, we are getting an error for not having double equals here:

```
  Line 78:22:   Expected '!==' and instead saw '!='  eqeqeq
```

Can we apply the following change to get rid of it?

```suggestion
        if (password !== repeatPassword) {
```

**Comment by kaankabalak** [comment]
Another minor feedback, can we have 2 spaces instead of 4 spaces for this component? Thanks in advance!

**Comment by dvaldivia** [comment]
@kaankabalak I'm going to address comments on #316

**Pull request merged**